release-notes/v1_67.md

```diff
@@ -181,6 +181,45 @@ The [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscj
 
 ![Debugger for Java supporting lazy variable resolution during a debugging session](images/1_67/java-lazy-variable.gif)
 
+
+### Jupyter
+
+**Web Extension**
+
+The Jupyter extension can now be installed as a web extension for use at [vscode.dev](https://vscode.dev). Functionality is currently quite limited with support only for notebooks (.ipynb files) being run on Jupyter servers that do not require https.
```

Comment:
Helpful to make this a link to the Jupyter extension on the Marketplace so that there's no ambiguity

Reply:
Someone who hasn't been reading our release notes regularly wouldn't have context so it's helpful to provide a link so they don't need to search on the web or Extensions view